src/services/optimizations/fpsManager.ts

```diff
@@ -30,7 +30,11 @@ class FpsManager {
 
       this.averageFPS = Math.floor(sum / 10);
 
-      window.requestAnimationFrame(refreshLoop);
+      if (typeof window === "undefined") {
+        global.setTimeout(refreshLoop, 16);
```

Comment:
Please don't ask about this magic number, it's taken from library code :)

https://github.com/gravity-ui/graph/blob/main/src/lib/Scheduler.ts#L3